src/schema/v2/artworkMediumTypes.ts

```diff
@@ -0,0 +1,12 @@
+import { GraphQLList, GraphQLFieldConfig } from "graphql"
+import { ResolverContext } from "types/graphql"
+import AttributionClass from "./artwork/mediumType"
```

Comment:
kind of an odd naming for the default import. (sorry if i'm being a doofus around attribution class vs medium :((

Reply:
No, this was totally a mistake on my end! Fixed in the follow-up PR. Another downside of default exports.